content/src/fieldConfig/business-formation.json

```diff
@@ -242,6 +262,23 @@
         "radioYesText": "Yes",
         "error": "Select 'Yes' or 'No'."
       },
+      "isVeteranNonprofit": {
+        "label": "Non-Profit Veteran Corporation",
+        "description": "Is your Domestic Non-Profit a `Veteran Corporation|veteran-corporation`?",
+        "radioNoText": "No",
+        "radioYesText": "Yes",
+        "error": "Select 'Yes' or 'No'.",
+        "domNpVetCorpReviewText": "**Yes**, my Domestic Non-Profit is a Veteran Corporation.",
+        "yesBoardMembersReviewText": "**Yes**, my Non-Profit will have board members.",
+        "noBoardMembersReviewText": "**No**, my Non-Profit will not have board members.",
+        "undefinedBoardMembersReviewText": "**My Non-Profit will or will not have board members**",
+        "boardMembersQualificationsReviewText": "**Board member qualifications are specified**",
+        "rightsAndLimitationsReviewText": "**Rights and limitations of the** board **members are specified**",
+        "choosingTrusteesReviewText": "**The method for choosing trustees are specified**",
+        "distributingAssetsReviewText": "**The method for distributing assets are specified**",
```

Comment:
[pebble] For these 5 keys where the entire string is bold, this violates the "I have not added any markdown to labels, titles and button text in config" item. If this text should be bold always, then we should control that in the code with a <strong> tag, not here in the markdown.

The ones that are only partially bold like `"**Yes**, my Non-Profit will have board members.` are fine.

Reply:
Changed

Reply:
Still not fixed yet, @cbrunefearless.

Reply:
Change has been made.. just need unit tests to pass.